src/code/z_skelanime.c

```diff
@@ -1007,7 +1007,7 @@ void AnimationContext_SetLoadFrame(PlayState* play, PlayerAnimationHeader* anima
 
     if (entry != NULL) {
         PlayerAnimationHeader* playerAnimHeader = Lib_SegmentedToVirtual(animation);
-        void* ram = (void*)frameTable;
```

Comment:
Hmm replacing this temp with a pad and just using frameTable in the Dma function seems to work. Not 100% sure it is better, though I do think we should rename the temp variable if we do.

Reply:
Sounds better, I'll go with it